packages/devtools/src/launcher.js

```diff
@@ -120,6 +120,8 @@ function launchBrowser (capabilities, product) {
 
     if (!executablePath) {
         throw new Error('Couldn\'t find executable for browser')
+    } else if (!executablePath.toLowerCase().includes(CHANNEL_FIREFOX_NIGHTLY)) {
+        throw new Error('Only Nightly release channel is supported in Devtools for Firefox')
```

Comment:
Let's add a longer error message explaining where you could download FF nightly and how to run it (setting firefox options).

Reply:
Sure. Will add it. And will fix the tests and add the unit tests too. Thanks!